CHANGES

```diff
@@ -1,3 +1,9 @@
+Current master
+-------------------
+- Allow staging and upload of data from a user's own buckets on Amazon
+  S3 (#). This uses boto3's `Bucket.copy()` feature. The caller must
+  ensure that AWS SDK credentials are properly configured.
```

Comment:
Do we need to expand on what "properly configured" means in this context? Probably not here but we should add something to `docs/uploads.md` explaining this feature and any potential gotchas.